INSTALL.md

```diff
@@ -58,40 +89,47 @@ If you have any additional plug-ins, copy/clone them into corresponding `src/plu
 :exclamation: **Reboot before executing the src/install.py** :exclamation:
 :exclamation: **You have to do the above steps before you do anything else** :exclamation:
 
-### Main-Install
+### Main installation
 
 :customs: **The installation script installs a lot of dependencies that may have different licenses**
 After reboot, enter the FACT_core directory and execute:
 
 ```sh
-$ src/install.py
+src/install.py
 ```
 
 :beer: **Get a drink... Installation of the dependencies might take some time...** :tea:
 
-More advanced setup options can be viewed using the help function of the installer `$ src/install.py --help` and are explained in some detailed in the following paragraphs.
+More advanced setup options can be viewed using the help function of the installer `src/install.py --help` and are 
+explained in some detailed in the following paragraphs.
 
 If the environment variable `FACT_INSTALLER_SKIP_DOCKER` is set the installer
 will skip all pulling/building of docker images.
 This is primarily used for the docker container of FACT but can also be used to
 save some time when you already have the images.
 
-## Multi system setup (**--backend**, **--frontend**, **--db**)
-
-The three components db, backend and frontend can be installed independently to create a distributed installation.
-
-The two worker components (frontend, backend) communicate exclusively through the database.
-The database in turn does not needed any knowledge of its place in the network, other than on which **ip:port** combination the database server has to be hosted.
-The config on the frontend system has to be altered so that the values of `common.postgres.server` and `common.postgres.port` match the **ip:port** for the database.
-The same has to be done for the backend.
-In addition, since the raw firmware and file binaries are stored in the backend, the `backend.firmware-file-storage-directory` has to be created (by default `/media/data/fact_fw_data`).
+## Multi-system (distributed) setup
```

Comment:
Why call it 'Multi-system (distributed)'?
Imo "Distributed setup" would sound nicer.

Reply:
I think so, too -> changed